jasmin_manage/serializers/service.py

```diff
@@ -84,10 +85,26 @@ class ServiceListSerializer(BaseSerializer):
     """
 
     requirements = ServiceRequirementSerializer(many=True)
+    # Add fields for summary data
+    is_active_requirements = serializers.SerializerMethodField()
```

Comment:
Possibly `has_active_requirements` might be more logical?

Reply:
Yep!